packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -18,22 +18,6 @@ propComponents:
   ]
 ---
 
-import {
-Button,
-DataList,
-DataListItem,
-DataListItemCells,
-DataListItemRow,
-DataListCell,
-DataListCheck,
-DataListAction,
-DataListToggle,
-DataListContent,
-Dropdown,
-DropdownPosition,
-KebabToggle,
-DropdownItem,
-} from '@patternfly/react-core';
 import CodeBranchIcon from '@patternfly/react-icons/dist/js/icons/code-branch-icon';
```

Comment:
Is there a reason you keep these imports from react-icons before the examples, but remove the react-core imports?

Reply:
All react-core and react-table imports are included in each react-live instance. There's no need to import them twice.